### ISSUE TITLE
Support for moded beacon blocks

### ISSUE THREAD

**Issue by SyncSteam** [ChloeDawn/BeaconOverhaul#9]
Can you create a config to add moded beacon blocks with different potencies?

(I'm assuming that beacon power is calculated by the total potency of all the blocks its made of per layer instead, of checking if the whole thing is the same block and then if it is, using the potency of that block. so you could mix blocks of the same potency and the beacon would still recognize that)

For example a full netherite beacon gives 3 potency, but the netherite stair from Blokus only acts as a normal beacon block. The stair still costs a full netherite block per stair, yet it wont provide any additional potency. Same can be said for the Aeternium block from BetterEnd, it has the same amount of netherite as a netherite block. 

Basically, I want my cool beacon to be powerful too. cuz rn it has the same power as a full iron one.
![image](https://user-images.githubusercontent.com/61635437/117715776-fabf0c00-b195-11eb-9dc6-96b545d69b61.png)

**Comment by ChloeDawn** [comment]
Currently, the way it behaves, is it works on minimum potency. So if you have a beacon made of low potency blocks (diamond) and replace one or two blocks with a high potency block (netherite) it will still be low potency. You would need to replace all blocks with high potency for it to be considered a high potency beacon. The simplest solution for mod support would be for me to add block tags for the additional tiers of potency. I'm just unsure how to describe them currently. Names such as `low_potency` and `high_potency` may be misleading as there would be a *lower* potency still, of blocks not in those two tags but still in `minecraft:beacon_base_blocks`.

**Comment by ChloeDawn** [comment]
Implemented in https://github.com/ChloeDawn/BeaconOverhaul/releases/tag/1.4.0+1.17 https://www.curseforge.com/minecraft/mc-mods/beaconoverhaul/files/3350334
